tasks/analysis.py

```diff
@@ -419,24 +419,26 @@ def process_impact_result(self, impact_result, analysis_id):
 
         # define the location of qgis template file
         custom_template_path = None
-        template_setting = settings.LOCALIZED_QGIS_REPORT_TEMPLATE
-        setting_template_path = template_setting.get(
-            analysis.language_code) or template_setting.get('en')
-        if os.path.exists(setting_template_path):
+        if analysis.custom_template:
             # resolve headless impact layer directory on django environment
-            filename = os.path.basename(setting_template_path)
-            dirname = os.path.basename(os.path.dirname(
-                impact_result['output']['analysis_summary']))
-            template_path = get_impact_path(os.path.join(
-                settings.GEOSAFE_IMPACT_OUTPUT_DIRECTORY, dirname, filename))
-            if os.path.exists(os.path.dirname(template_path)) and not (
-                    os.path.exists(template_path)):
-                shutil.copy(setting_template_path, template_path)
-                custom_template_path = template_path
+            filename = os.path.basename(analysis.custom_template)
+            dirname = os.path.dirname(
+                impact_result['output']['analysis_summary'])
```

Comment:
`impact_url` is enough I reckon. It's been defined before
```suggestion
                impact_url
```

Reply:
It's not url anymore, so we should change that.
Yeah, we can change into that.

Reply:
Uh, how do we change a suggestion? You have missing ) and I would like to put the line up one line.

Reply:
there is no multiline support yet :joy: